docs/docsite/rst/collections_guide/collections_installing.rst

```diff
@@ -114,6 +169,32 @@ Downloading a collection for offline use
 
 .. include:: ../shared_snippets/download_tarball_collections.txt
 
+.. _collection_local_install:
+
+Installing collections adjacent to playbooks
+--------------------------------------------
+
+You can install collections locally next to your playbooks inside your project instead of in a global location on your system or on your AWX.
```

Comment:
```suggestion
You can install collections locally next to your playbooks inside your project instead of in a global location on your system or via AWX.
```

Reply:
We should not use Latin in our docs. See https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#avoid-using-latin-phrases.

In this example, maybe just change 'on your AWX' to 'on AWX'.